### ISSUE TITLE
Ask For the problem of Loss Function

### ISSUE THREAD

**Issue by kkk-Lee** [GRAPH-0/RGTN-NIE#7]
Hello, I read your paper "Representation Learning on Knowledge Graphs for Node Importance Estimation" carefully, and saw that the paper mentioned that you applied multiple RMSE Loss for training. But in your code, I only see one MSE Loss in the geni_train.py file, would like to ask where did you set up the Loss Function mentioned in the paper so that I can understand your paper better.

**Comment by GRAPH-0** [comment]
GENI only use RMSE loss.
The loss function mentioned in the paper is [here](https://github.com/GRAPH-0/RGTN-NIE/blob/c73fe96d4b2c5490c273f8e487b1bb199afbdaf0/two_branch/model.py#L205).
